doc/release/RELEASE-NOTES.md

```diff
@@ -27,6 +27,7 @@ Move `countries` to a separate module so that it becomes possible to use both `d
 * [OSDEV-915](https://opensupplyhub.atlassian.net/browse/OSDEV-915) Upgrade Kafka tools to version 3.5.2
 
 ### Bugfix
+* [OSDEV-549](https://opensupplyhub.atlassian.net/browse/OSDEV-549) Facility Search. Search button overlaps dropdown items. Dropdown items in search were made  not to overlapping with button and containers in `Potential matches table` and `Find facility` search. The `isSideBarSearch` flag has been added to all search components to render properly regarding the place where the select is rendering.
```

Comment:
Double space between **made not** - > `Dropdown items in search were made  not`

Reply:
removed